source/ReproExamples/Issue514.tsx

```diff
@@ -1,66 +1,205 @@
+      <CustomizedSlider
         minimumValue={0}
-        maximumValue={7}
+        maximumValue={10}
         step={1}
         tapToSeek
-        onValueChange={setValue}
         minimumTrackTintColor={"#11FF11"}
         maximumTrackTintColor={"#11FF11"}
-        style={{ zIndex: 1 }}
+      />
+      <Text style={{ marginTop: 20 }}>
+        Slider just like before, but with image
+      </Text>
+      <CustomizedSlider
+        minimumValue={0}
+        maximumValue={10}
+        step={1}
         // eslint-disable-next-line
-        thumbImage={require("../ReproAssets/Issue346_ThumbImage.png")}
+        thumbImage={image}
+        tapToSeek
+        minimumTrackTintColor={"#11FF11"}
+        maximumTrackTintColor={"#11FF11"}
       />
-      <View
-        pointerEvents="none"
-        style={{
-          justifyContent: "space-between",
-          flexDirection: "row",
-          top: -25,
-          zIndex: 2,
-        }}
-      >
-        {options.map((i, index) => {
-          return (
-            <Fragment key={index}>
-              <View style={{ alignItems: "center" }}>
-                <SliderTrackMark
-                  key={`${index}-SliderTrackMark`}
-                  isTrue={value === i}
-                />
-                <Paragraph i={i} key={`${index}-Paragraph`} />
-              </View>
-            </Fragment>
+      <Text style={{ marginTop: 20 }}>
+        Slider with default steps indicator and numbers
+      </Text>
+      <CustomizedSlider
+        minimumValue={0}
+        maximumValue={10}
+        step={1}
+        tapToSeek
+        stepMarker
+        renderStepNumber
+        minimumTrackTintColor={"#123456"}
+        maximumTrackTintColor={"#123456"}
+      />
+      <Text style={{ marginTop: 20 }}>
+        Slider with customized indicator and no numbers
+      </Text>
+      <CustomizedSlider
+        minimumValue={0}
+        maximumValue={10}
+        step={1}
+        tapToSeek
+        stepMarker={(isMarked) => {
+          return isMarked ? (
+            <View style={styles.outerTrue}>
+              <View style={styles.innerTrue} />
+            </View>
+          ) : (
+            <View style={styles.outer}>
+              <View style={styles.inner} />
+            </View>
           );
-        })}
-      </View>
+        }}
+        minimumTrackTintColor={"#654321"}
+        maximumTrackTintColor={"#654321"}
+      />
+      <Text style={{ marginTop: 20 }}>
+        Slider with image as a thumb and no numbers
+      </Text>
+      <CustomizedSlider
+        minimumValue={0}
+        maximumValue={10}
+        step={1}
+        tapToSeek
+        stepMarker
+        // eslint-disable-next-line
+        thumbImage={image}
+        minimumTrackTintColor={"#654321"}
+        maximumTrackTintColor={"#654321"}
+      />
     </View>
   );
 };
 
-function SliderTrackMark({ isTrue }: TrackMarksProps) {
+function SliderTrackMark({ isTrue, thumbImage, StepMarker }: TrackMarksProps) {
```

Comment:
All those components naming will be adjusted later, in the actual implementation in Slider repository.
Can be kept here as is.